availability/main.py

```diff
@@ -13,35 +13,29 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-import flask
-from flask_helpers import routing
+import argparse
 
-from availability.api.v1 import api
-from availability.api.v1 import regions
-from availability import config
+from oss_lib import config
 
-
-app = flask.Flask(__name__, static_folder=None)
-app.config.update(config.get_config()["flask"])
-
-
-@app.errorhandler(404)
-def not_found(error):
-    return flask.jsonify({"error": "Not Found"}), 404
-
-
-for bp in [api, regions]:
-    for url_prefix, blueprint in bp.get_blueprints():
-        app.register_blueprint(blueprint, url_prefix="/api/v1%s" % url_prefix)
-
-
-app = routing.add_routing_map(app, html_uri=None, json_uri="/")
+from availability import app
+from availability import config as cfg
 
 
 def main():
-    app.run(host=app.config.get("HOST", "0.0.0.0"),
-            port=app.config.get("PORT", 5000))
-
-
-if __name__ == "__main__":
```

Comment:
Sure, you should start `availability-api` after installing the `availability` package because it [provides two console scripts](https://github.com/seecloud/availability/blob/master/setup.py#L24-L27): `availability-api` and `availability-watcher`.

You can create your own virtualenv and install `availability` into it:
```
git clone https://github.com/seecloud/availability.git
cd availability
virtualenv .venv
.venv/bin/pip install .
```

And then run it:
```
.venv/bin/availability-api --help
```

So, we can create a special target in `tox.ini` to simplify this preparation of venv and it will look like:

```
tox -e venv -- availability-api --help
```